themes/vuetober/src/components/Search.vue

```diff
@@ -48,6 +67,7 @@ export default {
   data() {
     return {
       searchTeacher: '',
+      isActive: -1,
```

Comment:
Toto by som skor nazval napriklad `activeTeacherId`. Pri `isActive` by som cakal ze to bude boolean

Reply:
Defaultna hodnota je `-1` z nejakeho dovodu? Skor by so tam dal `undefined` alebo `null`